### ISSUE TITLE
[A11y Fix] Correct TDP home : aria label mismatch

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1646]
***Description:***
Accessibility Insights fastpass &  @ttran-hub caught a WCAG 2.5.3 label in name issue for the top level home button in TDP labeled "TANF Data Portal" which is only labeled in aria as "Home"

This is a WCAG 2.1 violation, not 2.0, so it's not technically a Sec 508 relevant issue, but should probably be fixed (if needed) for Release 2. Thomas and Miles will sync to refine this ticket and target a fix such as an aria-label of "TANF Data Portal Home".

**AC**
- [ ] Make it like this ```<a href="/" aria-label="TANF Data Portal Home">TANF Data Portal</a>```

**Tasks**
- [x] Remove title attribute from "TANF Data Portal" header link 
- [x] Edit aria-label to read "TANF Data Portal Home" 

**Other Helpful Information** 
Succuss Criterion https://www.w3.org/WAI/WCAG21/Understanding/label-in-name
Compliant examples https://www.w3.org/WAI/WCAG21/Techniques/general/G208

**Comment by robgendron** [comment]
@reitermb is this still needed?
